src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -82,7 +83,10 @@ export default function applyAuthInterface(httpClient, authConfig) {
     try {
       decodedToken = jwtDecode(cookies.get(httpClient.accessTokenCookieName));
     } catch (error) {
-      // empty
+      /* istanbul ignore next */
+      if (httpClient.loggingService && httpClient.loggingService.logError) {
+        httpClient.loggingService.logError(error);
```

Comment:
Until we know this is an error, can you do something like the following:
```
httpClient.loggingService.logInfo(`Error decoding JWT token: ${error}`, {jwtDecodError: error});
```
The changes include:
1. Use `logInfo`, because we don't yet know this is an issue.
2. Adding error into customAttributes may not be necessary, but did it just in case we wanted it separated out.